src/issues.rs

```diff
@@ -83,6 +83,8 @@ pub enum IssueType {
     InvalidStopLocationTypeInTrip,
     /// The parent station of this stop is not a valid one
     InvalidStopParent,
+    /// The Id is not in ASCII encoding
+    IdNotAscii,
```

Comment:
Il faudrait l'ajouter dans le README https://github.com/etalab/transport-validator#list-of-checks-done-by-the-validator